internal/backend/remote/backend_context.go

```diff
@@ -27,8 +27,9 @@ func (b *Remote) LocalRun(op *backend.Operation) (*backend.LocalRun, statemgr.Fu
 	var diags tfdiags.Diagnostics
 	ret := &backend.LocalRun{
 		PlanOpts: &tofu.PlanOpts{
-			Mode:    op.PlanMode,
-			Targets: op.Targets,
+			Mode:     op.PlanMode,
+			Targets:  op.Targets,
+			Excludes: op.Excludes,
```

Comment:
By browsing the code, looks like this isn't really needed, as in any call of the `LocalRun` method there is never `Targets` or `Excludes` on the operation.